internal/command/command.go

```diff
@@ -287,19 +287,21 @@ func checkVersion(logger output.Logger) {
 		return
 	}
 
-	latestVersion, ok := versions["stable"].(string)
+	latestVersionRaw, ok := versions["stable"].(string)
 	if !ok {
 		return
 	}
 
+	latestVersion := fmt.Sprintf("v%s", latestVersionRaw)
```

Comment:
What if the "v" already exists on it?

Reply:
It doesn't in the current homebrew API spec today, but yeah, probably doesn't hurt to check